source/loader/layers/sanitizer/asan_interceptor.hpp

```diff
@@ -154,8 +154,50 @@ struct ContextInfo {
     }
 };
 
-struct USMLaunchInfo {
-    LaunchInfo *Data = nullptr;
+struct AsanRuntimeDataWrapper {
+    AsanRuntimeData Host{};
+
+    AsanRuntimeData *DevicePtr = nullptr;
```

Comment:
please manage this pointer in AsanRuntimeDataWrapper

Reply:
Done